README.md

```diff
@@ -44,6 +57,150 @@ In most cases this command must be run by root user or through `sudo` command.
 
 BSD systems are not tested well. If ALSA works, Oto should work.
 
+## Usage
+
+The two main components of Oto are a `Context` and `Players`. The context handles interactions with
+the OS and audio drivers, and as such there can only be **one** context in your program.
+
+From a context you can create any number of different players, where each player is given an `io.Reader` that
+it reads bytes representing sounds from and plays.
+
+> Note that a single `io.Reader` must **not** be used by multiple players.
+
+### Playing sounds from memory
+
+The following is an example of loading and playing an MP3 file:
+
+```go
+package main
+
+import (
+    "github.com/hajimehoshi/go-mp3"
+    "github.com/hajimehoshi/oto/v2"
+    "time"
+    "os"
+)
+
+func main() {
+
+    //Read the mp3 file into memory
+    fileBytes, err := os.ReadFile("./my-file.mp3")
```

Comment:
os.Open? Or, the binary cannot be passed to mp3.NewDecoder, and actually it is not passed. Or do you mean using byte.Reader?

Reply:
Oh good catch!
Yes I meant to use a `bytes.NewReader`, which is now fixed.